src/platforms/ios/ioscontroller.swift

```diff
@@ -254,11 +256,55 @@ public class IOSControllerImpl : NSObject {
             }
         }
     }
+    
+    /**
+    * @brief Enables / Disables the Always-On Rule on the runnel
+    * This is used for the "start-on-boot" feature. 
+    * We are setting a rule matching * any network change
+    * which will cause the phone to start the VPN after the phone boots.
+    *
+    * Note: For the Setting to be applied to the iOS settings: 
+    *       tunnel!.saveToPreferences needs to be called.  
+    * @param setEnabled: bool If the rule should be added or removed.
+    **/
+    @objc func setAlwaysOn(setEnabled: Bool){
+        if(!setEnabled){
+            tunnel!.isOnDemandEnabled = false;
+            tunnel!.onDemandRules = []
+            return;
+        }
+        let alwaysConnect = NEOnDemandRuleConnect()
+        alwaysConnect.interfaceTypeMatch = .any
+        tunnel!.isOnDemandEnabled = true
+        tunnel!.onDemandRules = [alwaysConnect]
+    }
+    
+    @objc func getAlwaysOn() -> Bool{
```

Comment:
```suggestion
    @objc var getAlwaysOn: Bool {
```

Personal preference, but IMO a bit cleaner - making this a computed var. Feel free to ignore if you want.